BikeBattle/app/src/main/java/edu/hm/cs/bikebattle/app/activities/MainActivity.java

```diff
@@ -121,6 +132,29 @@ public void error(int error, Throwable exception) {
   }
 
   @Override
+  public void refreshUserInfo() {
+
+    final User user = getPrincipal();
+    final String name = user.getName();
+    final String email = user.getEmail();
+    final Uri foto = getUserPhoto();
+
+    runOnUiThread(new Runnable(){
```

Comment:
des mit dem abstract in den anderen klassen macht kein sinn ich weiß schon aber das ich in der baseactivity dann die ganzen textfelder deklarieren muss macht noch weniger Sinn. 
Und das mit dem runnable bin ich nur so auf die aktuellen Daten gekommen